src/main/java/mate/academy/model/ShoppingCart.java

```diff
@@ -14,7 +15,7 @@
 public class ShoppingCart {
     @Id
     private Long id;
-    @OneToMany
+    @OneToMany(cascade = CascadeType.REMOVE)
     private List<Ticket> tickets;
```

Comment:
let's better remove this for now, and do it by hands
CascadeType.REMOVE is controversial